flink-connector/src/test/java/com/datastrato/gravitino/flink/connector/hive/TestHivePropertiesConverter.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *  Copyright 2024 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.flink.connector.hive;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.apache.flink.configuration.Configuration;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class TestHivePropertiesConverter {
+
+  private static final HivePropertiesConverter CONVERTER = HivePropertiesConverter.INSTANCE;
+
+  @Test
+  public void testToGravitinoCatalogProperties() {
+    Configuration configuration =
+        Configuration.fromMap(
+            ImmutableMap.of(
+                "hive-conf-dir", "src/test/resources/flink-tests", "flink.bypass.key", "value"));
+    Map<String, String> properties = CONVERTER.toGravitinoCatalogProperties(configuration);
```

Comment:
`toGravitinoCatalogProperties` load properties from `hive-conf-dir`, it's better to check the loaded properties.